master/rock.py

```diff
@@ -313,15 +313,51 @@ def Build(factory):
         env={'AUTOBUILD_CACHE_DIR': CACHE_IMPORT_DIR},
         ifReached="build")
 
+class ReportPathRender:
```

Comment:
https://github.com/buildbot/buildbot/blob/ec38d8f7bbffa4fc792d5bb6466c336015131197/master/buildbot/test/util/properties.py#L22

What's this for?

Reply:
No idea. Maybe check that the class matches the interface ?